include/clap/ext/audio-ports.h

```diff
@@ -26,8 +27,11 @@ enum {
    // Main port must be at index 0.
    CLAP_AUDIO_PORT_IS_MAIN = 1 << 0,
 
-   // The prefers 64 bits audio with this port.
-   CLAP_AUDIO_PORTS_PREFERS_64BITS = 1 << 1,
+   // The port can be used with 64 bits audio
+   CLAP_AUDIO_PORT_SUPPORTS_64BITS = 1 << 1,
+
+   // 64 bits audio is preferred with this port
+   CLAP_AUDIO_PORTS_PREFERS_64BITS = 1 << 2,
```

Comment:
In which situation would this preference be useful? Wouldn't hosts either use 32-bit audio internally and pass that audio to plugins as is, or use 64-bit audio in their audio graph and either convert it to and from 32-bit for plugins that don't support 64-bit audio or pass that to plugins as is? The host converting its own 32-bit audio to 64-bit for a plugin wouldn't be very useful, in that case the plugin could just do the conversion themselves (and save on a buffer memcpy).

Reply:
Maybe the host would instead produce a 64 bit audio in certain situation instead.
Also imagine that you put the plugin in a modular system, you can then have the info that the port prefers 64 bit audio and then connect a 64 bit stream to it. We have situations like this in Bitwig's device builder.

Reply:
Wouldn't that be incredibly inefficient? You either have a 64-bit stream, or you don't. If the audio data was originally 32-bit (since there's not really any reason to use 64-bit audio on the audio graph level), then the plugin would be much better off doing the conversion to 64-bit itself if it really needs 64-bit audio internally.

Reply:
#70 also asks about this, so I'll repeat the question there. Please ignore this thread!

Reply:
In bitwig device builder, there are components which requires 64 bit audio on certain ports because they process the input in way were the precision matters.
When we're patching cables, we get the visual info that this port wants 64 bit audio, and this hint helps us to make some decision.
So the info is also useful at the human level.